### PR TITLE
added human species overview

### PR DESCRIPTION
**Pull request by bleehu**
modified:   01_00_Species_Overview.txt

linking issue #762

### CODE REVIEW

File: CharacterCreation/01_00_Species_Overview.txt [hunk 1]
```diff
@@ -36,6 +36,22 @@ Species Overview
 
 ===== Humans =====
 
+    Humans are bipedal organic mammals. They origionally hail from a formerly
```
Comment (Reveraine) [inline CharacterCreation/01_00_Species_Overview.txt]: SP: originally

**Review by Reveraine — COMMENTED** [review]
I like it. I think it could use a little more refining in the middle, as the same type of paragraph is not present in the Opaleite blurb, but I don't have any good ideas on how to proceed as of yet. I'll chew in it for a while.

File: CharacterCreation/01_00_Species_Overview.txt (hunk 1 above)
Comment (rosebrallier) [inline CharacterCreation/01_00_Species_Overview.txt]: Suggestion: "Mammals" implies "organic"; we could delete the latter.

File: CharacterCreation/01_00_Species_Overview.txt [hunk 2]
```diff
@@ -36,6 +36,22 @@ Species Overview
...
+blue-green planet called Earth, and now make up the majority of the Terran 
+Empire. Humans are hearty, enterprising and resourceful. 
+
+    Humans can excell at any class. You could be a grizzled human soldier who's
```
Comment (rosebrallier) [inline CharacterCreation/01_00_Species_Overview.txt]: Sp: Excel
Gr: as any class
Suggestion: "who has" has better cadence and clearer meaning than "who's." (Who's most often refers to "who is.")

File: CharacterCreation/01_00_Species_Overview.txt [hunk 3]
```diff
@@ -36,6 +36,22 @@ Species Overview
...
+seen more than their fair share of combat. Or you could be the charismatic 
+gunslinging galatic bounty hunter who always gets their man, but rarely gets the
+girl. You could be the intuitive human engineer who takes care of her drones 
+like they were her beloved pets. Or the recent med student grad who's a whiz
```
Comment (rosebrallier) [inline CharacterCreation/01_00_Species_Overview.txt]: Gr: "are her beloved pets, or the..."
Suggestion: "med school" or "recently graduated doctor"

**Review by rosebrallier — APPROVED** [review]
No major issues here! Just a few suggestions before we ship.

**Pull request merged**